### PR TITLE
feat: add notification center

### PR DESCRIPTION
**Pull request by gondar00**
fixes: #3470

### CODE REVIEW

File: frontend/src/_components/NotificationCenter/Notification.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import moment from 'moment';
+import { commentsService } from '@/_services';
+
+export const Notification = ({ id, creator, comment, updatedAt, commentLink, isRead, fetchData }) => {
+  const getComment = (comment) => {
+    var regex = /(\()([^)]+)(\))/g;
```
Comment (TheLearneer) [inline frontend/src/_components/NotificationCenter/Notification.jsx:7]: Isn't it better not to use `var`?

File: server/src/services/comment.service.ts [hunk 2]
```diff
@@ -88,6 +91,50 @@ export class CommentService {
...
+          id: notification.id,
+          creator,
+          comment: notification.comment.comment,
+          createdAt: notification.comment.createdAt,
+          updatedAt: notification.comment.updatedAt,
+          commentLink,
+          isRead: notification.isRead,
+        };
+      });
+      return Promise.all(_notifications);
```
Comment (gsmithun4) [inline server/src/services/comment.service.ts:132]: should use await to resolve all `_notifications`

File: server/src/services/comment.service.ts [hunk 3]
```diff
@@ -124,6 +171,16 @@ export class CommentService {
     return foundComment;
   }
 
+  public async updateCommentUser(commentUserId: string, body: any) {
+    const item = await this.commentUsersRepository.update(commentUserId, { ...body });
+    return item;
+  }
+
+  public async updateAllCommentUser(body: any) {
+    const item = await this.commentUsersRepository.update({}, { ...body });
```
Comment (gsmithun4) [inline server/src/services/comment.service.ts:180]: please pick and pass the updatable params, passing entire body will cause security issues

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/src/services/comment.service.ts (hunk 2 above)
Comment (gondar00) [reply server/src/services/comment.service.ts:132]: async/await implicitly returns promise,

```
return Promise.all(_notifications);
```
is same as 
```
return await Promise.all(_notifications);
```

File: server/src/services/comment.service.ts (hunk 3 above)
Comment (gondar00) [reply server/src/services/comment.service.ts]: got it, I've updated

[the remark this answers is not in the document]
Comment (gondar00) [reply server/src/services/comment.service.ts]: Added FindCondition

**Comment by shubh22** [comment]
Task linked: [CU-2h1aq0e Create comment notifications](https://app.clickup.com/t/2h1aq0e)

**Pull request merged**